pkg/resource/aws/testdata/acc/aws_security_group/.terraform.lock.hcl

```diff
@@ -5,6 +5,7 @@ provider "registry.terraform.io/hashicorp/aws" {
   version     = "3.19.0"
   constraints = "3.19.0"
   hashes = [
+    "h1:+7Vi7p13+cnrxjXbfJiTimGSFR97xCaQwkkvWcreLns=",
```

Comment:
Is it on purpose ?

Reply:
It is yes, didn't want to make a separate PR to fix this little line ...